bcda/auth/alpha_test.go

```diff
@@ -101,78 +99,6 @@ func (s *AlphaAuthPluginTestSuite) TestDeleteClient() {
 	assert.Empty(s.T(), aco.AlphaSecret)
 }
 
-func (s *AlphaAuthPluginTestSuite) TestGenerateClientCredentials() {
-	r, err := s.p.GenerateClientCredentials("", 0)
-	assert.Equal(s.T(), auth.Credentials{}, r)
-	assert.NotNil(s.T(), err)
-
-	aco := models.ACO{
-		UUID: uuid.NewRandom(),
-		Name: "Gen Client Creds Test",
-	}
-	err = connections["TestGenerateClientCredentials"].Save(&aco).Error
-	assert.Nil(s.T(), err, "wtf? %v", err)
-	// we know that we use aco.UUID as the ClientID
-	clientID := aco.UUID.String()
-
-	r, err = s.p.GenerateClientCredentials(clientID, 720)
-	assert.Equal(s.T(), auth.Credentials{}, r)
-	assert.Contains(s.T(), err.Error(), "have a registered client")
-
-	// quick and dirty register client
-	aco.ClientID = aco.UUID.String()
-	err = connections["TestGenerateClientCredentials"].Save(&aco).Error
-	assert.Nil(s.T(), err, "wtf? %v", err)
-	email := fmt.Sprintf("%s@genclientcredstest.com", testUtils.RandomHexID())
-	user, err := models.CreateUser("Fake User", email, aco.UUID)
-	assert.Nil(s.T(), err, "wtf? %v", err)
-
-	r, err = s.p.GenerateClientCredentials(clientID, 0)
```

Comment:
As long as we are going to leave in the method signatures I think we should be running the tests on them.  I think this line can stay and just check that an error comes back.